CDP4Orm/MigrationScript/EngineeringModel_20210109_6_4_0_0_Model_Updates.sql

```diff
@@ -1,4 +1,4 @@
-﻿-- Create table for class LogEntryChangelogItem (which derives from: Thing)
+-- Create table for class LogEntryChangelogItem (which derives from: Thing)
```

Comment:
am i to understand that you are changing existing migrations?

Reply:
no, i think he copied the old ones unmodified to Old_WithViews folder and made a new set here by modifying the old ones. However @antoineatrhea I also don't get exactly why this is needed?

Reply:
I kept the old ones to have a traceability. But it's required to still have those migrations since the structure is on version 1.1.27, so we have to apply model updates migration. However, I can not keep the exact same as before since we don't have VIEWS and Data functions anymore. I added a new migration that cleans every VIEWS and Data functions to be sure to not have of them if we have an old database